python/test/unit/language/test_core.py

```diff
@@ -5227,6 +5240,10 @@ def compute_scratch_buffer_shape(src_layout, dst_layout, shape):
 def test_convert2d(M, N, src_layout, interm_layout, dst_layout, dtype, device, tmp_path: pathlib.Path):
     if str(src_layout) == str(dst_layout):
         pytest.skip()
+    if (isinstance(src_layout, DotOperandLayout)
+            and isinstance(interm_layout, SharedLayout)) or (isinstance(dst_layout, DotOperandLayout)
+                                                             and isinstance(interm_layout, SharedLayout)):
+        pytest.skip("DotOperandLayout <-> SharedLayout conversion is not completely supported")
```

Comment:
Is this still the case? If so, why?

Reply:
We do support it in the code but we didn't turn it on. It still goes through `SharedToDotOperandMMAv2orV3.cpp` to use ldmatrix.

See the task here https://github.com/orgs/triton-lang/projects/4/views/1?pane=issue&itemId=85582604

BTW, I've also dropped a task for you :)

https://github.com/orgs/triton-lang/projects/4/views/1?pane=issue&itemId=85583148

Reply:
Ah, right, saw it! It's in my local stack of tasks (i.e. written down in my notebook lol)